uk_bin_collection/uk_bin_collection/councils/SouthamptonCityCouncil.py

```
@@ -22,9 +22,31 @@ def parse_data(self, page: str, **kwargs) -> dict:
+            "sec-fetch-dest": "document",
+            "sec-fetch-mode": "navigate",
+            "sec-fetch-site": "same-origin",
+            "upgrade-insecure-requests": "1",
+            "user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/141.0.0.0 Safari/537.36",
+        }
+
+        params = {
+            "UPRN": {user_uprn},
+        }
+
+        r = requests.get(
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)